package.json

```diff
@@ -41,6 +41,7 @@
     "standard": "^16.0.2"
   },
   "scripts": {
+    "prepare": "husky install",
```

Comment:
Why `husky install` is needed:
https://blog.typicode.com/husky-git-hooks-autoinstall/